google-bigquery/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -10,7 +10,7 @@ For information about compatibility and fixed issues, refer to the Google BigQue
 
 To use this connector, you must be familiar with:
 
-* The Google BigQuery Connector’s API
+* The Google BigQuery API
 * Anypoint Connectors
```

Comment:
should we add those reference changes to 1.0 too?

Reply:
@sathya0 Yes, it's in this PR.